x-pack/plugin/sql/qa/src/main/resources/geo/docs.csv-spec

```diff
@@ -10,9 +10,18 @@
 
 selectAsWKT
 // tag::aswkt
-SELECT city, ST_ASWKT(location) location FROM "geo" WHERE city = 'Amsterdam';
+SELECT city, ST_AsWKT(location) location FROM "geo" WHERE city = 'Amsterdam';
 
      city:s    |     location:s
 Amsterdam      |point (4.850311987102032 52.347556999884546)
 // end::aswkt
 ;
+
+selectWKTToSQL
+// tag::wkttosql
+SELECT CAST(ST_WKTToSQL('POINT (10 20)') AS STRING) location;
```

Comment:
we might be able to get away from this. For intervals, I've basically changed the way we do assertion where the actual type is checked and if it's of a custom type, a custom serialization takes place to check the strings.
In other words there would be no casting, CSV would read the result as a String but when the check is done, the geo type is detected, the result casted as the string and the two strings asserted.

No need to hold the PR for this.

Reply:
Cool. I will definitely update this test when it lands in master.